test/src/utils/fs.spec.ts

```diff
@@ -106,23 +108,28 @@ describe("UTILS fs tests suites", () => {
     describe("cloneRepos(...)", () => {
```

Comment:
I would move this `"cloneRepos(...)` describe block to top of testing scope to match the structure of functions in `fs.ts`

Reply:
Thanks for looking into It but it is not important the order of the nested `describe()` function